src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -33,13 +33,13 @@
 
     public static final String ERROR_UPDATE_NON_EXISTENT = "Trying to update non-existent Feedback Session : ";
 
-    public List<FeedbackSessionAttributes> getAllOpenFeedbackSessions(Date startUtc, Date endUtc) {
+    public List<FeedbackSessionAttributes> getAllOpenFeedbackSessions(Instant rangeStart, Instant rangeEnd) {
         List<FeedbackSessionAttributes> list = new LinkedList<>();
 
         Calendar startCal = Calendar.getInstance();
-        startCal.setTime(startUtc);
+        startCal.setTime(TimeHelper.convertInstantToDate(rangeStart));
         Calendar endCal = Calendar.getInstance();
-        endCal.setTime(endUtc);
+        endCal.setTime(TimeHelper.convertInstantToDate(rangeEnd));
```

Comment:
Can you refactor line 46 and 47 as well so that it does not use the deprecated `convertToUserTimeZone`? I don't think we should have the Calendar type here. We should do everything in `Instant` and have 1 final step that convert the `Instant` to `Date` before passing to a library method that needs a `Date`, (Objectify's `filter` in this case). Then when Objectify finally support `Instant`, we can just delete the last conversion.